src/components/sbb-divider/sbb-divider.stories.js

```diff
@@ -0,0 +1,105 @@
+import { h } from 'jsx-dom';
+import readme from './readme.md';
+import {
+  SbbColorBlackDefault,
+  SbbColorWhiteDefault,
+} from '@sbb-esta/lyne-design-tokens/dist/js/sbb-tokens';
+
+const documentationPlatformContainerStyle = (context) => {
+  if (context.args.negative) {
+    return {};
+  }
+
+  return `background-color: ${SbbColorWhiteDefault};`;
+};
+
+const wrapperStyle = (context) => {
+  setTimeout(() => {
```

Comment:
You could just wrap the examplte template in a div with a height. That would be easier.

Reply:
Done 👍